test/proto/BUILD

```diff
@@ -80,39 +146,47 @@ java_proto_library(
 scalapb_proto_library(
     name = "test_proto_java_conversions",
     visibility = ["//visibility:public"],
-    with_flat_package = True,
-    with_java = True,
     deps = [
         ":test2",
-        ":test_proto_java_lib",
         "//test/proto2:test",
     ],
 )
 
 scalapb_proto_library(
     name = "test_proto",
     visibility = ["//visibility:public"],
-    with_grpc = True,
-    deps = [":test_service"],
+    deps = [
+        ":blacklisted_proto",
+        ":test_service",
+        "@io_bazel_rules_scala//test/proto:other_blacklisted_proto",
+    ],
 )
 
-scala_proto_srcjar(
```

Comment:
can we no longer directly exercise srcjar?

Reply:
Nope, we could bring it back if it was useful, though since the aspect generates one internally they would be duplicates of some form